ros/src/waypoint_loader/waypoint_loader.py

```diff
@@ -37,8 +37,8 @@ def new_waypoint_loader(self, path):
     def quaternion_from_yaw(self, yaw):
         return tf.transformations.quaternion_from_euler(0., 0., yaw)
 
-    def get_velocity(self, velocity):
-        return velocity/3.6
+    def kmph2mps(self, velocity_kmph):
+        return (velocity_kmph * 1000) / (60 * 60)
```

Comment:
Nice! I would however be careful using integer numbers. If `velocity_kmph` is an integer, then all the operations will be performed on integers. So it would be better to replace 1000 with 1000.0 and 60 with 60.0.

We simply have it like this:

    return velocity_kmph * 0.27778

It's not a magic number anymore because the function clearly indicates what's happening (some comment can also be added to further clarify).

Reply:
Ah good point! Will update now.